test/e2e/app-dir/app/index.test.ts

```diff
@@ -41,6 +41,35 @@ createNextDescribe(
           middlewareManifest.functions['/slow-page-with-loading/page'].regions
         ).toEqual(['sfo1'])
       })
+
+      it('should have correct maxDuration values in manifest', async () => {
+        const middlewareManifest = JSON.parse(
+          await next.readFile('.next/server/middleware-manifest.json')
```

Comment:
Was talking with @feedthejim and I think we should migrate this manifest to a more generic `functions-manifest.json` so that it makes more sense for general function configs.

Reply:
@ijjk yeah I've raised this to Jimmy too, but went ahead with those changes since existing data being written in this file is not semantically correct either. Would it be cool to ship it like this and then hand the file name migration process off to the Next.js team?